source/installation-guide/wazuh-dashboard/step-by-step.rst

```diff
@@ -123,16 +123,131 @@ Starting the Wazuh dashboard service
+
+      curl -k -X PUT "https://localhost:55000/security/users/2" -H "Authorization: Bearer $TOKEN" -H 'Content-Type: application/json' -d' 
+      {
+        "password": "SuperS3cretPassword!"
+      }'
+
+   .. code-block:: console
+     :class: output   
+
+      {"data": {"affected_items": [{"id": 2, "username": "wazuh-wui", "allow_run_as": true, "roles": [1]}], "total_affected_items": 1, "total_failed_items": 0, "failed_items": []}, "message": "User was successfully updated", "error": 0}
+
+   See the :doc:`Securing the Wazuh API </user-manual/api/securing-api>` section for additional security configurations. 
+
+   .. note:: Remember to store these passwords securely. 
+
+#. In your `Wazuh dashboard` server, update ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` with your new password.  
+
+   .. code-block:: yaml
+     :emphasize-lines: 6
+    
+      hosts:
+        - default:
+          url: https://localhost
+          port: 55000
+          username: wazuh-wui
+          password: SuperS3cretPassword!
+          run_as: false 
+
+#. Restart the Wazuh dashboard.
+
+   .. include:: /_templates/common/restart_dashboard.rst
+
+
+
+Change the default Wazuh indexer credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command on any Wazuh indexer node to change all the default Wazuh indexer passwords. 
+
+#. Use the Wazuh password tool to change all the internal users passwords. 
+
+   .. code-block:: console
+  
+     # /usr/share/wazuh-indexer/plugins/opensearch-security/tools/wazuh-passwords-tool.sh --change-all
+  
+   .. code-block:: console
+     :class: output
+
+     INFO: The password for user admin is o5XWBs044S5PJ1edYgw4R4MOM7r00Hjm
+     INFO: The password for user kibanaserver is mqhbijWWw8vVyuOBDtFQvqoLYwMdrcXP
+     INFO: The password for user kibanaro is jTbFvrCSQ4LaLmcNtcFOzVUvCL24nYtN
+     INFO: The password for user logstash is bmtERx0schYGoANyGiWnyed6044CYGQv
+     INFO: The password for user readall is j9JOe7nEhKZhjyfUWVXhI2FNaM5A7eT6
+     INFO: The password for user snapshotrestore is tYyAqG9U72RURf0svrvJ8rtiVEzqjdmg
+     INFO: The password for user wazuh_admin is NHJMpiJkeBgjwNSJnoPV1KD9XMD7a98N
+     INFO: The password for user wazuh_user is N364F1kSkgKfUkVPzRxnCKwszaDVLqu0
+     WARNING: Passwords changed. Remember to update the password in the Wazuh dashboard and Filebeat nodes if necessary, and restart the services.
+
+   .. note:: Remember to store these passwords securely. 
+
+If you have an all-in-one deployment, the ``wazuh-passwords-tool.sh`` automatically updates the credentials in your Filebeat and Wazuh dashboard keystores. For a distributed deployment, you need to update them manually following the steps below. 
+
+**Only for distributed deployments**
+
+#. On your `Wazuh server`, update the `admin` password in the Filebeat keystore. Replace ``<admin-password>`` with the random password generated in the previous step and run the following command:  
```

Comment:
```suggestion
#. On your `Wazuh server`s, update the `admin` password in the Filebeat keystore. Replace ``<admin-password>`` with the random password generated in the previous step and run the following command:  
```
? Not sure, but how we make understand that this action must be completed in all wazuh servers?

Reply:
Done. The instruction now states: "On your Wazuh servers, update the admin password in the Filebeat keystore."